pango/src/layout.rs

```diff
@@ -0,0 +1,52 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use glib::translate::*;
+
+use crate::LayoutLine;
+
+/// The result of [`LayoutLine::x_to_index`].
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
+pub struct HitPosition {
+    /// The UTF-8 byte offset of the grapheme closest to the position.
+    ///
+    /// This position is relative to the start of the line, not the start
+    /// of the [`Layout`]'s text.
+    ///
+    /// [`Layout`]: crate::Layout
+    index: i32,
```

Comment:
Just realized that you need to make those fields public, otherwise the type will be useless.